libs/constants/index.js

```diff
@@ -83,7 +83,7 @@ const scriptSrcValues = [
   "assets.codepen.io",
   "production-assets.codepen.io",
 
-  "'sha256-x6Tv+AdV5e6dcolO0TEo+3BG4H2nG2ACjyG8mz6QCes='", // inline no flicker
+  "'sha256-mr0WqMazG99/aIjMg/7cYeaiijSgAspY8Ssvrg63DF0='", // inline no flicker
```

Comment:
@fiji-flo I used https://csplite.com/csp/sha/ to calculate this hash, but I'm not very confident that this one is right. How did you generate the hash?